{"uuid":"75a28ce9-2c36-4e28-9429-b93690fc2da2","historyId":"26a961230328bb88d9e7ff8bc2c836c3","fullName":"tests.InventoryTest.VerifyWrapMenuAbout","labels":[{"name":"package","value":"tests.InventoryTest"},{"name":"testClass","value":"tests.InventoryTest"},{"name":"testMethod","value":"VerifyWrapMenuAbout"},{"name":"parentSuite","value":"SauceDemo Suite"},{"name":"suite","value":"InventorySuite - Firefox"},{"name":"subSuite","value":"tests.InventoryTest"},{"name":"host","value":"DESKTOP-F37PN0H"},{"name":"thread","value":"36060@DESKTOP-F37PN0H.TestNG-tests-1(37)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Checks if 'About' button in Wrapper menu is functional"},{"name":"epic","value":"Inventory Module - TC_I"},{"name":"feature","value":"Inventory functionality checks"},{"name":"severity","value":"normal"}],"links":[],"name":"VerifyWrapMenuAbout","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"About button is not functioning expected [true] but found [false]","trace":"java.lang.AssertionError: About button is not functioning expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat tests.InventoryTest.VerifyWrapMenuAbout(InventoryTest.java:119)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java)\r\n\tat --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)\r\n\tat java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:132)\r\n\tat java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:113)\r\n\tat java.base/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:274)\r\n\tat org.testng.internal.thread.ThreadUtil.execute(ThreadUtil.java:70)\r\n\tat org.testng.SuiteRunner.runInParallelTestMode(SuiteRunner.java:442)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Test verifies if 'About' button works","steps":[],"attachments":[{"name":"Failure Screenshot","source":"185fcd3a-ede2-44f8-a5c6-ec30b8e04c41-attachment.png","type":"image/png"}],"parameters":[{"name":"browser","value":"firefox"}],"start":1755603222984,"stop":1755603223142}